src/main/webapp/index.html

```diff
@@ -43,6 +43,10 @@
           <span class="sr-only">Loading...</span>
         </div>
       </div>
+      <div id="error-surfacer" style="display:none">
+        <h4 id="error-display-text"style="color:rgb(255, 99, 71);"> Something went wrong :( </h4>
+        <h5 id="error-details"style="color:rgb(255, 99, 71);"><h5>
```

Comment:
Maybe after :( we could say ", try another link!"

Reply:
Good idea!